app/assets/stylesheets/custom.css.scss

```diff
@@ -4,10 +4,80 @@ body {
   @include outer-container
 }
 
+.hero {
+  ///////////////////////////////////////////////////////////////////////////////////
+  $base-border-radius: 3px !default;
+  $base-accent-color: #477DCA !default;
```

Comment:
Color `#477DCA` should be written as `#477dca`<br>Each selector in a comma sequence should be on its own line